qcfractal/cli/qcfractal_manager.py

```diff
@@ -148,6 +150,7 @@ def parse_args():
     # Additional args
     optional = parser.add_argument_group('Optional Settings')
     optional.add_argument("--test", action="store_true", help="Boot and run a short test suite to validate setup")
+    optional.add_argument("--ntests", type=int, help="How many tests per found program to run, does nothing without --test set")
```

Comment:
Default to one here?

Reply:
It does

https://github.com/MolSSI/QCFractal/pull/215/files#diff-d72c13bbbcebb1df47f9d2d3f874c558R61